deploy/crd/clustercompliancereports.crd.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
To simply the installation we have a single file deploy/static/starboard.yaml that should include these CRDs. You can update and use hack/update-starboard.yaml.sh script to concatenate installed resources.